openlibrary/plugins/openlibrary/events.py

```diff
@@ -30,5 +24,4 @@ def on_new_version(self, page):
 def setup():
     """Installs handlers for various events.
     """
-    eventer.bind("page.edit.libraries", on_library_edit)
     eventer.bind("page.edit", on_page_edit)
```

Comment:
This _looks_ like it could be removed, along with the `def on_page_edit(page)` -- I wouldn't know if this causes other problems. Is there a specific reason you left it with a `pass`, or just general caution?

Reply:
Mainly to leave a vestige of the mechanism in place in case someone wanted to use it again. I'm OK with removing the whole thing if you think that's best.